goserver/session/session.go

```diff
@@ -0,0 +1,28 @@
+// Package session implements server session.
+package session
+
+import (
+	"encoding/json"
+	"time"
+)
+
+// Session stores necessary information for a telehealth session.
+type Session struct {
+	ID        string    `json:"id"`
+	FHIRURL   string    `json:"fhir_url"`
+	ExpiresAt time.Time `json:"expires_at"`
+}
+
+// Bytes converts the session to JSON bytes.
+func (s *Session) Bytes() ([]byte, error) {
```

Comment:
After some efforts to use protobuff, I figured it will introduce complexity in building/deploying and I don't think it's worthy it for serializing a simple struct.